internal/generate/crd/crd.go

```diff
@@ -371,6 +374,26 @@ func setCRDStorageVersion(crd *apiextv1beta1.CustomResourceDefinition) {
 	crd.Spec.Versions[0].Storage = true
 }
 
+func setSDKstamps(crd *apiextv1beta1.CustomResourceDefinition, operatorType string) {
+	annotations := crd.GetAnnotations()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+	addStampsToAnnotations(annotations, operatorType)
+	crd.SetAnnotations(annotations)
+}
+
+func addStampsToAnnotations(mapAnnotations map[string]string, operatorType string) {
```

Comment:
```suggestion
func (g Generator) addStampsToAnnotations(mapAnnotations map[string]string) {
```
same here ^

Reply:
This _could_ be a method, but is more easily unit-tested if it isn't.